docs/migration-console.md

```diff
@@ -0,0 +1,123 @@
+# Migration Console
+
+## Introduction
+
+**Overview**
+The Migration Console serves as the control plane for orchestrating and managing the components of the Migration Assistant. It coordinates the activities of various components in the migration pipeline and provides a single point of interaction for users. The Migration Console can refer both to the ECS instance deployed in the user's account and selected VPC, and the command line application installed on that instance to interact with the Migration Assistant components.
+
+**Goals and Scope**:
+
+- Ensure centralized control and visibility into the migration process.
+- Provide interfaces for managing communication between the Migration Console and other components.
+- Operate statelessly with a transparent source of truth for predictable and configurable behavior.
+
+## System Architecture
+
+The Migration Console is deployed into the customer's VPC and maintains connections with effectively all other components of the Migration Assistant. The below diagram is not comprehensive and does not illustrate any of the data flow between various other components.
+
+![Migration Solution Architecture](diagrams/migration-console-arch.svg)
+[Source](https://tiny.amazon.com/jmayhucf/desia2zIMi)
+
+### Individual Connections
+
+#### Bootstrap Box
```

Comment:
The bootstrap box isn't part of the console, but part of the solutions deployment, seems like it shouldn't be in this doc?